packages/ckeditor5-list/src/documentlist/documentlistediting.js

```diff
@@ -134,25 +139,29 @@ export default class DocumentListEditing extends Plugin {
 			// First we want to allow user to outdent all indendations from other features then he can oudent list item.
 			outdent.registerChildCommand( commands.get( 'outdentList' ), { priority: 'lowest' } );
 		}
+
+		// Register conversion and model post-fixer after other plugins had a chance to register their attribute strategies.
```

Comment:
Note for Kuba: "Głupia pętla" :P

Reply:
Unfortunately, the problem is that GHS is registering conversions after all the plugins got initialized (at the `DataController#init` event) and that is too late for document lists to register converters. We could postpone the DL conversion (and post-fixing) registration but that would very complicate the code. So I went with the convention of prefixing document list model attributes (`list`) so those can be captured by the prefix itself (when needed, the conversion itself is using the later registered conversion strategies because they are really used when any of those attributes need conversion).